detect_test.go

```diff
@@ -0,0 +1,102 @@
+package static
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/paketo-buildpacks/packit/v2"
+	"github.com/paketo-buildpacks/packit/v2/scribe"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+)
+
+func testDetect(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect = NewWithT(t).Expect
+
+		workingDir string
+		buffer     *bytes.Buffer
+		name       = "static-confgen"
+		indexFile  = `
+<!DOCTYPE html>
+<html></html>`
+	)
+
+	it.Before(func() {
+		buffer = bytes.NewBuffer(nil)
```

Comment:
nitpicking: a `bytes.Buffer` can just be used when declared as a variable (see last sentence of https://pkg.go.dev/bytes#NewBuffer).

Reply:
If you have multiple test cases, this makes sense since the buffer is automatically cleared before every new test case. For example in `detect_test.go` it makes it a bit cleaner as you don't need to manually initialize it in every context.

Reply:
Yeah makes sense. Thx for the explanation.